jellyfin_apiclient_python/api.py

```diff
@@ -466,7 +468,11 @@ def send_request(self, url, path, method="get", timeout=None, headers=None, data
         LOG.debug(request_settings['timeout'])
         LOG.debug(request_settings['headers'])
 
-        return request_method(url, **request_settings)
+        try:
+            return request_method(url, **request_settings)
+        except requests.exceptions.RequestException as error:
+            LOG.error(error)
```

Comment:
It seems as if doing this would mean the method would return `None` instead of throwing an exception, which may cause problems.

Reply:
Yes, agreed, for an inner method like send_request I think allowing the exception to occur is okay.
Would you agree that it then should be caught in functions validate_authentication_token, get_public_info, check_redirect and utc_time?

Reply:
As long as it keeps the API the same. If users of the API rely on catching exceptions from the API calls, I don't think we should break that.

Reply:
Right, it would break that. I would say that it makes the API more user-friendly if it handles exceptions and the user just has to verify whether the result they get is what is expected. But maintaining backward compatibility is definitely also a strong plus.